src/examples/forms/input-group/Disabled.tsx

```diff
@@ -17,7 +17,9 @@ const Example = () => (
         <Label>Plant name generator</Label>
         <InputGroup>
           <Input value="Sheepberry" disabled />
```

Comment:
Tied to the comment about the button copy, maybe we shouldn’t show this field as filled out in the demo.

Reply:
updated.